core/components/molecules/tabs/__tests__/Tabs.test.tsx

```diff
@@ -133,7 +133,7 @@ describe('Tabs component', () => {
 
```

Comment:
Add tests for Tab as children of Tabs and content in Tab, inline content and tabs without activeIndex in props.